instrumentation/testing/testing.go

```diff
@@ -154,6 +161,57 @@ func (test *Test) End() {
 	test.span.Finish()
 }
 
+func (test *Test) extractTestLoggerOutput() {
+	output := extractTestOutput(test.t)
+	if output == nil {
+		return
+	}
+	outStr := string(*output)
+	var logArray []map[string]interface{}
```

Comment:
We can use a `[]log.Field` of `opentracing` directly here

Reply:
done